Makefile

```diff
@@ -1437,7 +1447,7 @@ librocksdb_env_basic_test.a: $(OBJ_DIR)/env/env_basic_test.o $(LIB_OBJECTS) $(TE
 	$(AM_V_at)$(AR) $(ARFLAGS) $@ $^
 
 db_bench: $(OBJ_DIR)/tools/db_bench.o $(BENCH_OBJECTS) $(TESTUTIL) $(LIBRARY)
-	$(AM_LINK)
+	$(AM_LINK_WHOLE)
```

Comment:
Same question as above.  Why not have db_bench depend on LIBRARY + PLUGIN_LIBRARIES?